docs/source/index.rst

```diff
@@ -13,6 +13,7 @@ This package provides classes and functions for the analysis of water dynamics a
    :caption: Contents:
 
    getting_started
+   examples
```

Comment:
add a section after examples about *output*

Reply:
The original contents of the output section were put in to the examples subsections.

Reply:
Make it a separate section — it's important enough.

Reply:
Moved